.github/workflows/prepare-transaction.yml

```diff
@@ -14,13 +14,14 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-        with:
-          sparse-checkout: |
-            .github/scripts/validate_new_chain_request.sh
-          sparse-checkout-cone-mode: false
-
       - name: Run the validation script
-        run: bash ./.github/scripts/validate_new_chain_request.sh
+        run: |
+          RPC_URL=$(echo "$ISSUE_BODY" | grep -E -o 'https?://[^ ]+' -m 1 | head -1)
+          SUMMARY_FILE=summary.json RPC=$RPC_URL npm run verify:new-chain-request
```

Comment:
We should potentially add `summary.json` to the `.gitignore` in case we run this exact line locally.

Reply:
Updated 👍🏾